client/src/dbus/mod.rs

```diff
@@ -59,3 +59,6 @@ pub use error::{Error, ServiceError};
 pub use service::Service;
 mod collection;
 pub use collection::Collection;
+
+pub const XDG_SCHEMA: &str = "xdg:schema";
```

Comment:
Both should probably be inside the AsAttributes trait with a comment explaining what it is useful for